### ISSUE TITLE
java.lang.NoClassDefFoundError on start 1.13.1

### ISSUE THREAD

**Issue by i0xHeX** [VolmitSoftware/React#452]
Log:
```
[12:36:13] [Server thread/INFO]: [React] Enabling React v6.573
[12:36:14] [Server thread/WARN]: java.lang.NoClassDefFoundError: net/minecraft/server/v1_13_R1/World
[12:36:14] [Server thread/WARN]: 	at com.volmit.react.controller.FeatureController.start(FeatureController.java:73)
[12:36:14] [Server thread/WARN]: 	at com.volmit.react.ReactPlugin.onEnable(ReactPlugin.java:83)
[12:36:14] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264)
[12:36:14] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:339)
[12:36:14] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:403)
[12:36:14] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:427)
[12:36:14] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:341)
[12:36:14] [Server thread/WARN]: 	at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:214)
[12:36:14] [Server thread/WARN]: 	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:698)
[12:36:14] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[12:36:14] [Server thread/WARN]: Caused by: java.lang.ClassNotFoundException: net.minecraft.server.v1_13_R1.World
[12:36:14] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:105)
[12:36:14] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:100)
[12:36:14] [Server thread/WARN]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[12:36:14] [Server thread/WARN]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[12:36:14] [Server thread/WARN]: 	... 10 more
[12:36:14] [Server thread/INFO]: React: Language: enUS
[12:36:14] [Server thread/INFO]: 1.13
[12:36:14] [Server thread/INFO]: 1.13 Detected, Safe mode activated.
```
Spigot 1.13.1
React 6.573

**Comment by NextdoorPsycho** [comment]
noted third error

**Comment by NextdoorPsycho** [comment]
Fixed in 6.575
